source/api_documentation/network_integration/affiliates/index.rst

```diff
@@ -47,33 +47,79 @@ Similar to advertisers, you are not allowed to delete if one or more campaigns e
     - The site name that matches site id_from_network.
 
   * - **users**
-    - json array of hashes
-    - 0 or more users for the organization. Each must have the first 5 fields below.
+    - JSON array of hashes
+    - 0 or more users for the organization. Each must have first 5 fields below.
 
   * - id_from_network
     - string
-    - The network object_id for this user.
+    - The network id for this User.
+
+  * - email_settings
+    - JSON array of hashes
+    - Each hash has two required fields:
+
+      ``email_address``: string in RFC 2822 addr-spec format. The user’s email address. Unique for this user.
+
+      ``use_for_notifications``: boolean used to indicate if notifications should be sent to the email address.
 
-  * - email_address
-    - string in RFC 2822 addr-spec format
-    - The user’s email address. Unique for this user.
+      A user must have at least one email address where ``use_for_notifications`` is true.
 
   * - first_name
-    - string(must not be blank)
+    - string (Required)
     - The user’s first name.
 
   * - last_name
-    - string(must not be blank)
+    - string (Required)
     - The user’s last name.
 
-  * - phone_number
+  * - contact_phone_number
     - string in ITU E.164 format or 10-digit US form (no punctuation)
     - The user’s phone number.
 
+  * - oauth_refresh_token
+    - string
+    - Not used. Reserved.
+
   * - role
     - One of: Super (default), Manager, Member, Observer
     - This user’s role in this organization. (A user may have different roles in different organizations)
 
+  * - notify_on_budgets
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_campaign_applications
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_campaign_expirations
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_creative_duplication_requests
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_network_announcements
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_performance_notifications
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_monthly_campaign_performance_reports
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_weekly_campaign_performance_reports
+    - boolean, optional, defaults to false
+    -
+
+  * - notify_on_call_activities
+    - boolean, optional, defaults to false
+    -
+
```

Comment:
shouldn't you document `can_login_via_platform` ?

```
- can_login_via_platform
- boolean (read-only)
- Indicates if user can directly login with username and password. If false, the user is managed via SSO.
```

Reply:
I did initially but did not mark it as read-only (didn't realize that was a pattern) and removed it because it seemed like the table was for editable request params. I now see the read-only option in there as well. I'll add it back in. Thanks for catching that.